stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2628,196 +2658,156 @@ final internal class _Native${Self}StorageImpl<${TypeParameters}> {
     _fixLifetime(self)
   }
 
-  internal init(_doNotCallMe: ()) {
+  override internal init(_doNotCallMe: ()) {
     _sanityCheckFailure("Only initialize these by calling create")
   }
 }
 
-@_fixed_layout
-public // @testable
-struct _Native${Self}Storage<${TypeParametersDecl}> :
-  _HashStorage, CustomStringConvertible {
-  internal typealias Owner = _Native${Self}StorageOwner<${TypeParameters}>
-  internal typealias StorageImpl = _Native${Self}StorageImpl<${TypeParameters}>
+// TODO: conditionally conform to _HashStorage for better compile-time safety.
+// TODO: conditionally conform to CustomStringConvertible.
```

Comment:
I didn't add these. I'm pretty sure the first one is complaining that we aren't ever asserting NativeStorage: HashStorage, because that only holds when K: Hashable. Not sure why CustomStringConvertible is interesting.

The whole HashStorage thing is kinda dubious and would be removed with eager bridging anyway, though. 

Delete the comments?